From: Pietro Battiston <me@pietrobattiston.it>
Date: Wed, 21 Dec 2011 15:29:24 +0100
Subject: rerun-cython

Shapely is distributed with the .pyx Cython file already transformed into a .c
file. This patch imports the modified "build_ext" from the Cython Distutils,
and then feeds it the .pyx (rather than .c) file.
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 328c0b6..0b5f17f 100644
--- a/setup.py
+++ b/setup.py
@@ -12,7 +12,7 @@ except:
 from distutils.errors import CCompilerError, DistutilsExecError, \
     DistutilsPlatformError
 from setuptools.extension import Extension
-from setuptools.command.build_ext import build_ext
+from Cython.Distutils import build_ext
 from setuptools import setup, find_packages
 import sys
 import platform
@@ -109,7 +109,7 @@ elif (hasattr(platform, 'python_implementation')
 else:
     ext_modules = [
         Extension("shapely.speedups._speedups", 
-              ["shapely/speedups/_speedups.c"], libraries=['geos_c']),
+              ["shapely/speedups/_speedups.pyx"], libraries=['geos_c']),
     ]
 
 try:
